README.md

```diff
@@ -471,6 +509,20 @@ CMS are a practical way to setup a website with many features. CMS often come wi
 
 See https://staticsitegenerators.net and https://www.staticgen.com
 
+
+## Ticketing systems
+*Web-based ticketing system.*
```

Comment:
Yes, this is even better.

 * Split section into
  - [ ] Software Dev/Bug Trackers
  - [ ] Ticketing
 - [ ] Link Software Dev/Bug Trackers -> Ticketing
 - [ ] Link Ticketing -> Software Dev/Bug Trackers